functions-methods.js

```diff
@@ -33,4 +57,25 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (emailadres3){
+
+    const lengthEmail = emailadres3.length-1;
+    if (emailadres3.includes(",") === false && emailadres3.includes("@") === true && emailadres3.lastIndexOf(".") !== lengthEmail ){
+        console.log(true)}
+    else {
+        console.log(false)
+    }
```

Comment:
Je zou dit nog wat kunnen versimpelen
```suggestion
    console.log(!emailadres3.includes(",") && emailadres3.includes("@") && emailadres3.lastIndexOf(".") !== lengthEmail);
```